Documentation/conf.py

```diff
@@ -0,0 +1,212 @@
+# -*- coding: utf-8 -*-
```

Comment:
This means that python is now a project dependency. The container dependencies must be updated in the `Doxkerfile`s. 

That and all Sphnx related dependencies should be added as well.

Reply:
All required dependencies are installed by `./Tools/generate_api_doc.sh `. Reason this is done in the script and not inside docker is to keep the docker build small. Any tools and linter dependencies are installed by the respective scripts themselves when run the first time.

Reply:
Script that generates the documentation below install all the required dependencies.

Reply:
Ok, I didn’t catch that.